libcst/tests/test_pyre_integration.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-strict
+
+import json
+import subprocess
+from pathlib import Path
+from typing import Tuple
+
+from mypy_extensions import TypedDict
+
+
+def _run_command(command: str) -> Tuple[str, str]:
+    process = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True
+    )
+    stdout, stderr = process.communicate()
+    return stdout.decode(), stderr.decode()
+
+
+class Position(TypedDict):
+    line: int
+    column: int
+
+
+class Location(TypedDict):
+    path: str
+    start: Position
+    stop: Position
+
+
+class InferredType(TypedDict):
+    location: Location
+    annotation: str
+
+
+def _sort_by_position(data: InferredType) -> Tuple[int, int, int, int]:
+    start = data["location"]["start"]
+    stop = data["location"]["stop"]
+    return (start["line"], start["column"], stop["line"], stop["column"])
+
+
+if __name__ == "__main__":
```

Comment:
Where is the actual test being run?  This looks like it's generating data based off the python files in `libcst/tests/pyre`

Reply:
@mvismonte you read it right, the actually unit test was just added now.
It's `PyreIntegrationTest`. Currently, Pyre don't return typing on a couple cases and I've report them to Shannon.

Reply:
Running test_pyre_integration.py is a codegen. (We also have codegen for other LibCST code).
We run it manually and commit the generated code. In this case, the generated code is `simple_class.json` which is the output of Pyre query with some trans.